### ISSUE TITLE
Run "task sync" on loading / exiting buffer

### ISSUE THREAD

**Issue by abers** [tbabej/taskwiki#224]
Unsure if this is already possible already, couldn't see anything in the help file, but it would be useful if could setup taskwiki to run "task sync" on before loading and after exiting buffers with taskwiki viewports.

**Comment by tbabej** [comment]
No, this is not part of current functionality. I can see how this is helpful, however, it could slow down the user experience (`task sync` can be particularly slow on bad internet connections).

I could see this added if somebody is willing to contribute a small patch, however it should be optional and disabled as default

**Comment by ludenticus** [comment]
To sync tasks every time you open `vimwiki` or leave vim/nvim, add this to your `.vimrc`:

```vim
augroup TaskWiki
  autocmd!
  autocmd BufEnter vimwiki :TWSync
  autocmd VimLeave * :TWSync
augroup END
```
It assumes you have installed the `vim-taskwarrior` plugin.
